common_gaku_dependencies.rb

```diff
@@ -1,12 +1,10 @@
-source 'http://production.s3.rubygems.org'
+source 'http://rubygems.org'
 
 group :test do
   gem 'capybara',                 '~> 2.1'
-  gem 'selenium-webdriver',       '~> 2.39'
+  gem 'selenium-webdriver',       '2.43'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.